apps/homescreen/build/default-homescreens.json

```diff
@@ -0,0 +1,34 @@
+{
+  "homescreens": [
+    [
+      ["apps", "communications", "dialer"],
+      ["apps", "sms"],
+      ["apps", "communications", "contacts"],
+      ["apps", "email"],
+      ["apps", "search"]
+    ], [
+      ["apps/collection/collections", "social"],
+      ["apps/collection/collections", "games"],
+      ["apps/collection/collections", "music"]
+    ], [
+      ["apps", "camera"],
+      ["apps", "gallery"],
+      ["apps", "music"],
+      ["apps", "video"],
+      ["apps", "marketplace.firefox.com"]
+    ], [
+      ["apps", "calendar"],
+      ["apps", "clock"],
+      ["apps", "settings"],
+      ["apps", "fm"]
+    ], [
+      ["hosted_apps", "bugzillalite"],
+      ["hosted_apps", "facebook"],
+      ["hosted_apps", "twitter"],
+      ["apps", "costcontrol"]
+    ]
+  ],
+  "preferences": {
+    "grid.cols": 3
+  }
+}
```

Comment:
No good reason, but I wanted the file to look familiar to whoever's responsible for updating it - I should perhaps remove the collections section, but it's nice to keep it there to test that it won't fall over if it encounters something it doesn't handle.